Source/town.cpp

```diff
@@ -130,9 +130,9 @@ void TownCloseHive()
 	SetDungeonMicros();
 }
 
-/**
- * @brief Update the map to show the closed grave
- */
+/** 
+* @brief Update the map to show the closed grave
+*/

```

Comment:
Indentation is wrong and there is some trailing white space (see code style), please make sure that lines you did not intend to change do not show up as changed.
```suggestion
/**
 * @brief Update the map to show the closed grave
 */
```